server/ocsp.go

```diff
@@ -103,6 +116,39 @@ func (oc *OCSPMonitor) getCacheStatus() ([]byte, *ocsp.Response) {
 	return oc.raw, oc.resp
 }
 
+func (oc *OCSPMonitor) getLocalStatus() ([]byte, *ocsp.Response, error) {
+	opts := oc.srv.getOpts()
+	storeDir := opts.StoreDir
+	if storeDir == _EMPTY_ {
+		return nil, nil, fmt.Errorf("store_dir not set")
+	}
+
+	key := fmt.Sprintf("%x", sha256.Sum256(oc.Leaf.Raw))
+
+	oc.mu.Lock()
+	raw, err := ioutil.ReadFile(filepath.Join(storeDir, key))
```

Comment:
missing `defaultOCSPStoreDir` in the join

Reply:
Thanks, should be fixed now